allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_data_transforms.py

```diff
@@ -126,8 +126,9 @@ def get_running_speed(self, lowpass=True) -> RunningSpeed:
             raise DataFrameIndexError(
                 f"Expected index to be named 'timestamps' but got "
                 f"'{running_data_df.index.name}'.")
-        return RunningSpeed(timestamps=running_data_df.index.values,
-                            values=running_data_df.speed.values)
+        return pd.DataFrame({
+            "timestamps": running_data_df.index.values,
+            "values": running_data_df.speed.values})
```

Comment:
One simplification we might consider is to remove the `get_running_speed` method in `BehaviorOphysDataXforms` and instead have this function check if there is a `sync_file` in order to perform time rebasing.

This has the advantage that getting running speed would come from one method between `BehaviorDataTransforms` and `BehaviorOphysDataTransforms`. The downside is that this 'Behavior' only method would be referring to things (the sync file in this case) that would exist for a 'BehaviorOphys' experiment

Reply:
food for thought. not this week perhaps